src/navigation/router.js

```diff
@@ -8,59 +8,86 @@ import SymptomScreen from '../symptoms/symptoms-screen';
 import Reminder from '../reminder/reminder';
 import Export from '../export/export';
 import DailyChart from '../calendar/daily-chart';
-import {Platform} from 'react-native';
+import {AppState, Platform} from 'react-native';
 import Notes from '../survey/notes-screen';
 import Onboarding from '../onboarding/onboarding';
 import CGU from '../legal/cgu-screen';
 import Privacy from '../legal/privacy-screen';
 import LegalMentions from '../legal/legal-mentions-screen';
+import matomo from '../services/matomo';
 
 const Stack = createStackNavigator();
 
-const Router = () => {
-  return (
-    <NavigationContainer>
-      <Stack.Navigator initialRouteName="tabs" headerMode="none">
-        <Stack.Screen name="tabs" component={Tabs} />
-        {availableData.map((surveyItem, index) => (
+class Router extends React.Component {
+  async componentDidMount() {
+    await matomo.initMatomo();
+    matomo.logAppVisit();
+    AppState.addEventListener('change', this.onAppChange);
+  }
+
+  componentWillUnmount() {
+    matomo.logAppClose();
+    AppState.removeEventListener('focus', this.onAppChange);
+  }
+
+  appState = AppState.currentState;
```

Comment:
```suggestion
  const appState = AppState.currentState;
```

Reply:
Il est préférable d'utiliser var/let/const pour déclarer une variable : https://www.tutorialsteacher.com/javascript/javascript-variable#:~:text=It%20is%20Not%20Recommended%20to,anywhere%20in%20the%20web%20page.

Reply:
c'est une propriété, je ne pense pas que ce soit possible d'utiliser let/var/const